doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -202,8 +202,8 @@ An over use of host aggregates and availability zones can result in a granular p
 Recommendation: Separation of control zone and execution zone into different security zones
 
 <a name="3.3.4"></a>
-### 3.3.4. Flavour management (i.e. T-Shirt type)
-Content to be developed
+### 3.3.4. Flavor management
+A flavor defines the compute, memory, and storage capacity of nova instances. When instances are spawned, they are mapped to flavors which define the available hardware configuration for them. For simplicity, the flavors can be named according to T-shirt sizes like XS,S,M,L,XL,2XL,3XL and so on. The specifications for these sizes should map to the predefined compute flavors lister [here](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4211-predefined-compute-flavours).
```

Comment:
@msidana Since we are asking the use of the CNTT flavors as specified in RM Chapter 4, cn you use them as examples (.tiny, .small, .medium, ...)?

More importantly, since the OpenStack Newton release there have been no predefined flavors. Do you think it would thus be worth mentioning how the CNTT pre-defined public flavors can be created by Admin users?

Reply:
Ok, I can specify them as .tiny, .small etc to be in sync with RM.
Regarding your second question:
From architecture perspective, I don't think we should mention about **HOW** the flavor is created. An operator may define these flavors via several options like some automation, manual creation, post deployment step etc.

Reply:
@msidana the section title is "Flavor Management", but the current content is silent on the topic.  If we are not to discuss Flavor Management then what would be added beyond what is in the Glossary and RM.

Reply:
Yes, you have a point. The title doesn't justifies the description written underneath and vice-versa. I can think about 2 options
1. Rename the title and talk about flavors in general as being done in other sections of this chapter for other concepts.
2. We can write something about flavor management. But being a RA, I'm not convinced that we should talk about flavors management - like creation, deletion, metadata, public/private access etc. This is already defined and well documented in OpenStack.
What do you suggest out of these 2 OR any other better alternative ?

Reply:
@msidana on your second point about not defining how - could we provide some pointer to the reader... better we define these things consistently using one method in OpenStack as then we have a level playing field with no surprises...?   As to how that is mapped to different physical servers is variable but the software layer could be the same.

Reply:
we could say we recommend you do this using scheduler hints / host aggregates - I think we can take an opinionated approach.

Reply:
@msidana also on your comment, "Using NUMA, CPU pinning, threading etc will depend on the workload requirements...." but these are specified in the flavors and need to be covered in "Flavor Management."

Reply:
@pgoyal01 What changes do you suggest for this section ?